spec/game_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../game'
+
+describe 'Unit testing for Game Class' do
```

Comment:
Some of the tests fail. Kindly look into it and make sure that all your tests pass

![image](https://user-images.githubusercontent.com/38088818/230629085-25696377-fc2c-42ab-bb83-5c2ec3048eb8.png)